src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/EqualsTest.java

```diff
@@ -0,0 +1,367 @@
+                DoubleFilter doubleFilter = new DoubleFilter(id, name, description, productAttribute, min, max);
+                LongFilter longFilterWithDifferentType = new LongFilter(id, name, description, productAttribute, (long) min, (long) max);
+                assertNotEquals(doubleFilter,longFilterWithDifferentType);
+            }
+
+            @ParameterizedTest(name = "Time filter: {0},{1},{2},{3},{4},{5}")
+            @DisplayName("Double filters with different types: Time filter")
+            @CsvFileSource(resources = "DoubleFilter.csv", numLinesToSkip = 1)
+            void doubleFiltersWithDifferentTypesTimeFilter(int id, String name, String description, String productAttribute, double min, double max) {
+                DoubleFilter doublefilter = new DoubleFilter(id, name, description, productAttribute, min, max);
+                //We use Instant.MIN and Instant.Max instead of trying to convert a double value to an Instant value
+                TimeFilter timeFilterWithDifferentType = new TimeFilter(id, name, description, productAttribute, Instant.MIN, Instant.MAX);
+                assertNotEquals(doublefilter,timeFilterWithDifferentType);
+            }
+
+
+            @ParameterizedTest(name = "Different user min: {0},{1},{2},{3},{4},{5}")
+            @DisplayName("Double filters with different UserMin values")
+            @CsvFileSource(resources = "DoubleFilter.csv", numLinesToSkip = 1)
+            void doubleFiltersWithDifferentUserMinValues(int id, String name, String description, String productAttribute, double min, double max) throws IllegalMinMaxException {
+                DoubleFilter doubleFilter = new DoubleFilter(id, name, description, productAttribute, min, max);
+                DoubleFilter doubleFilterWithDifferentUserMin = new DoubleFilter(id, name, description, productAttribute, min, max);
+                doubleFilterWithDifferentUserMin.setUserMin(min+0.1);
+                assertNotEquals(doubleFilter,doubleFilterWithDifferentUserMin);
+            }
+
+            @ParameterizedTest(name = "Different user max: {0},{1},{2},{3},{4},{5} ")
+            @DisplayName("Double filters with different UserMax values")
+            @CsvFileSource(resources = "DoubleFilter.csv", numLinesToSkip = 1)
+            void doubleFiltersWithDifferentUserMaxValues(int id, String name, String description, String productAttribute, double min, double max) throws IllegalMinMaxException {
+                DoubleFilter doubleFilter = new DoubleFilter(id, name, description, productAttribute, min, max);
+                DoubleFilter doubleFilterWithDifferentUserMax = new DoubleFilter(id, name, description, productAttribute, min, max);
+                doubleFilterWithDifferentUserMax.setUserMax(max-0.1);
+                assertNotEquals(doubleFilter,doubleFilterWithDifferentUserMax);
+            }
+        }
+
+        @Nested
+        @DisplayName("Non-matching long filter test")
```

Comment:
Same as the naming concern for double filters

Reply:
Done